assets/src/common/helpers/index.js

```diff
@@ -199,3 +204,83 @@ export const getAspectRatioType = ( width, height ) => {
 		return 'square';
 	}
 };
+
+/**
+ * Gets the compiled template for a given notice message.
+ *
+ * @param {string} message The message to display in the template.
+ * @return {Function} compiledTemplate A function accepting the data, which creates a compiled template.
+ */
+export const getNoticeTemplate = ( message ) => {
```

Comment:
This function is simply extracted from the previous logic, and almost not changed at all.